agents/plugins/mk_oracle

```diff
@@ -2970,7 +2972,30 @@ for sid in $SIDS; do
 
     # MK_DB_CONNECT could be changed by do_custom_sqls!
     do_custom_sqls
+
+}
+
+ORA_TMPDIR="${MK_VARDIR}/tmp/mk_oracle"
+mkdir -p ${ORA_TMPDIR}
+TASK_NR=0
+for sid in $SIDS; do
+    skip=$(skip_sid "$sid")
+    if [ "$skip" == "yes" ]; then
+        logging "[${sid}] [local]" "Skipping this SID"
+        continue
+    fi
+    ((TASK_NR++))
+    logging "[${sid}] [local] starting background task '${TASK_NR}'"
+    (sid_actions $sid > "$ORA_TMPDIR/$sid") &
```

Comment:
"sid_actions" does not expect any arguments. Either change the signature of "sid_actions" or the call site.